build.sbt

```diff
@@ -177,5 +177,5 @@ addCommandAlias("fixCheckImports", "compile:scalafix --check SortImports; test:s
 // Scalafmt
 scalafmtOnCompile in ThisBuild :=
   sys.env
-    .get("CI")
+    .get("GITHUB_ACTIONS")
```

Comment:
Github has a built-in env variable, so I switched to using that since we no longer have a Travis config.

FYI, https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables